Contributing.md

```diff
@@ -1,19 +1,16 @@
-Contributing to Apollo Android GraphQL 
-======================================
+# Contributing to Apollo Android GraphQL
```

Comment:
Super Nit: I'd just go with `Apollo Android` for the project name (until it changes to something else maybe 👀 )